astro/src/content/docs/release-notes/index.mdx

```diff
@@ -44,6 +45,107 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
+  </Issue>
+* <Issue issue="2909">
+  The PHP client library is not handling libcurl errors gracefully, making it difficult to troubleshoot integration problems when using this library. See the [client library issue](https://github.com/FusionAuth/fusionauth-php-client/issues/28) for more details.
+  </Issue>
+* <Issue issue="2917">
+  When downloading login records from <Breadcrumb>System -> Login Records</Breadcrumb>, the exported file format contains a place for zip code, however the zip code values are not being populated in the export.
+  </Issue>
+* <Issue issue="2928" thanks="cezarneaga">
+  The `POST /api/user/registration` call is documented as returning a `refreshTokenId`, but this value is not being returned on the response.
+  </Issue>
+* <Issue issue="2932">
+  When editing a user's password in the FusionAuth admin UI after a new hashing scheme is set on the tenant, the password is not re-hashed using the new scheme. The re-hashing occurs as expected on a login or when the user changes their own password.
+  </Issue>
+
+### Enhancements
+* <Issue issue="1596" thanks="juansedo">
+  The `rate limit` error message was added to the default theme messages to make it more obvious that it is customizable. The `[RateLimitedException]` message key was previously supported, but not easily discoverable.
+  </Issue>
+* <Issue issue="1619">
+  Add an additional refresh token revocation policy to revoke a one-time use token on reuse. This policy helps protect against token theft by revoking the token if it were to be stolen and reused.
+  </Issue>
+* <Issue issue="2378">
+  FusionAuth can now accept encrypted SAML assertions when acting as a SAML service provider. Support for encrypted assertions when FusionAuth is the SAML identity provider was added in version `1.47.0 `.
+  </Issue>
+* <Issue issue="2537">
+  API keys can now be optionally set to expire at a given date/time. An expired key will not be deleted but will cause a `401` response to be returned when used. The expiration value can be edited to allow the expiration to be extended.
+  </Issue>
+* <Issue issue="2742">
+  Additional parameters are now accepted on the hosted backend `/app/login` and `/app/registration` endpoints. This means you can pass things like `login_hint`, `idp_hint`, and analytics parameters that will be available on the respective OAuth hosted pages.
+  </Issue>
+* <Issue issue="2859">
+  The First-time Setup wizard was improved with more descriptive and consistent text around using a Community plan license.
+  </Issue>
+* <Issue issue="2864">
+  In the new Webhook Event Log there were numerous small UX and copy improvements.
+  </Issue>
+* <Issue issue="2878">
+  Improved handling of a SAML `RelayState` in an IdP-initiated login. Previously, FusionAuth would only look for a valid ACS URL in the `RelayState`. Now, if the ACS URL can be resolved via other means, the `RelayState` value will be preserved and passed as a parameter in the final call to the ACS URL.
```

Comment:
I don't know how important the distinction is, but the trigger for passing along `RelayState` is that the value does not match a configured ACS URL (referred to as `Authorized redirect URLs` in the admin UI). However, this statement is correct in that it does require the ACS URL be resolved in another way, most likely by selecting the first configured option from the list.

Reply:
Let's leave this as-is and address it more specifically in the real docs.